java/mtsj/core/src/test/java/com/devonfw/application/mtsj/general/common/PermissionCheckTest.java

```diff
@@ -150,7 +153,7 @@ public void permissionFindOrdersByPost() {
   /**
    * It logs out after every test method.
    */
-  @After
+  @AfterEach
```

Comment:
in devon4j we provide test-base-classes. All inherit from `BaseTest` that allows to override `doSetUp()` and `doTearDown()`. The idea of these methods is that you avoid such migration issues and also mistakes when overriding without again placing the annotation. As MTS is a blueprint example we should follow our own guides here.